electron/electron.js

```diff
@@ -127,6 +142,7 @@ ipcMain.handle('save-output', async (event, savePath, outputBundles, saveLogs) =
     const outputFile = path.join(savePath, `mcode-extraction-patient-${data.index + 1}.json`);
     fs.writeFileSync(outputFile, JSON.stringify(data.bundle, null, 2), 'utf8');
   });
+  downloadBounceOnMac(savePath);
```

Comment:
For some reason this bounce animation doesn't occur for me when saving on the results page, but does occur when saving within the Config Editor. This is more of a nice to have than anything so it wouldn't hold up an approval, but it is strange because looking at the code I'm not sure why it would work in one location and not the other.

Reply:
When you are on the results page, are you saving to the Downloads folder? I know it' sa dumb question, but I just wanna make sure 😄

Reply:
No that's reasonable! I am saving to my downloads folder. I'm guessing you're not seeing the same behavior?

Reply:
I'm being honest with you, I didn't test completely and left that as an exercise to the reader 😅. I'll check to see what happens on my machine.

Reply:
Update: It does not in fact work on my machine